src/server/api/controllers/users.controller.js

```diff
@@ -33,26 +60,43 @@ const getUserById = async (id) => {
   return userById;
 };
 
+// SEARCH FOR A USER
 const getUsersByKeyword = async (searchWord) => {
   if (!searchWord) {
     throw new IncorrectEntryError('Use a keyword!', 400);
   }
 
-  const users = await knex('users').where(
-    'fullName',
-    'like',
-    `%${searchWord}%`,
-  );
+  const users = await client.search({
+    index: usersIndex,
+    body: {
+      query: {
+        multi_match: {
+          query: searchWord,
+          type: 'bool_prefix',
+          fields: ['fullName', 'fullName._2gram', 'fullName._3gram'],
```

Comment:
Here you should now use fullName.autocomplete because that is the subfield with the "search as you type" type.

Reply:
🤦🏽  spend so much time trying to go around it, not realizing or finding that it is just the autocomplete 😅
Done!